components/elm/src/biogeochem/PhosphorusDynamicsMod.F90

```diff
@@ -759,11 +759,41 @@ subroutine PhosphorusBiochemMin_balance(bounds,num_soilc, filter_soilc, &
         do j = 1,nlevdecomp
             biochem_pmin_to_ecosysp_vr_col(c,j)=0._r8
             do l = 1, ndecomp_pools
+<<<<<<< HEAD
```

Comment:
Looks like a conflict from the rebase still needs to be resolved.

Reply:
Yes, my rebase messed this up. Let me clean this.